src/components/Gallery/Gallery.scss

```
@@ -12,33 +12,30 @@
 	text-align: center;
   z-index: 999;
 
-  & img{
+  & .__image {
     margin-top: 5%;
     width: 500px;
   }
 
-  & .gallery-thumbnail{
+  & .gallery-thumbnail {

```

```suggestion
  .gallery-thumbnail {
```
When you are define somethinf inside a block, like:

```scss
.main {
  .item { }
}
```
Scss will compile to `.main .item`, so you don't need to put `&` before the `.item` selector